packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -98,6 +78,26 @@ export type DtmfTone =
   | 'Pound'
   | 'Star';
 
+/**
+ * Props for {@link Dialpad} component.
+ *
+ * @beta
+ */
+export interface DialpadProps {
+  strings?: DialpadStrings;
+  /**  function to send dtmf tones on button click */
+  onSendDtmfTone?: (dtmfTone: DtmfTone) => Promise<void>;
+  /**  Callback for dialpad button behavior*/
+  onClickDialpadButton?: (buttonValue: string, buttonIndex: number) => void;
+  /** set dialpad textfield content */
+  textFieldValue?: string;
+  /**  on change function for text field */
+  onChange?: (input: string) => void;
```

Comment:
Can we call it out here that this callback provides an unformatted plain text?

Reply:
updated